MvvmCross.Plugins/Email/Platform/iOS/MvxComposeEmailTask.cs

```diff
@@ -23,12 +25,12 @@ public class MvxComposeEmailTask
         : MvxIosTask
         , IMvxComposeEmailTaskEx
     {
-        private readonly IMvxIosModalHost _modalHost;
+        private readonly MvxIosViewPresenter _viewPresenter;
         private MFMailComposeViewController _mail;
 
         public MvxComposeEmailTask()
         {
-            _modalHost = Mvx.Resolve<IMvxIosModalHost>();
+            _viewPresenter = Mvx.Resolve<IMvxIosViewPresenter>() as MvxIosViewPresenter;
```

Comment:
This change introduces a new limitation: If you're using a ViewPresenter that doesn't inherit `MvxIosViewPresenter` you won't be able to use the plugin... Maybe we could do `UIApplication.SharedApplication.KeyWindow.RootViewController.PresentViewController(viewController, animated, completionHandler)` instead? We'll need some test cases on the TestProjects also.

Reply:
An alternative is to add `ShowModalViewController` to `IMvxIosViewPresenter`